app/src/main/AndroidManifest.xml

```diff
@@ -101,6 +101,16 @@
             android:name="androidx.work.impl.foreground.SystemForegroundService"
             android:foregroundServiceType="dataSync"
             tools:node="merge" />
+
+        <!-- Braze init -->
+        <service
+            android:name="com.braze.push.BrazeFirebaseMessagingService"
+            android:enabled="${fcmEnabled}"
```

Comment:
Is there an alternative flag we could use to enable it? 
I'm worried about potential conflicts when we integrate FirebaseMessagingService.
wdyt?

Reply:
Braze has provided a mechanism to resolve this conflict. We can implement that when time comes. Review `Using your own Firebase Messaging Service` under this [step](https://www.braze.com/docs/developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration/#step-1-register-braze-firebase-messaging-service).

OR

We can consider consolidating the FCM and Braze flags as we did in the `OpenEdXApp.kt` file. However, this approach would result in two separate services, which may not be feasible.

Reply:
Make sense, thank you!